### DISCUSSION TITLE
同步130万个文件共计980G的容量，复制到百度和阿里云盘出现的一些问题

### DISCUSSION THREAD

**Discussion by zts59** [AlistGo/alist#7249] [Q&A]
通过TAOSYNC或手动复制，
1、导致alist的DATA文件夹中的data.db会无限增大，把本地磁盘分区塞满，最大达到过299G。
2、可能是云盘的原因（受限），任务中的复制一直卡住，不会新增成功的记录（成功的记录删除后），也不会新增失败记录。杀进程终止后，用事先备份好的DATA.db替换后，启动重新复制，又有新的记录了。
3、希望有查看复制进度的日志，看看是什么原因，另希望终止全部任务的选项。

**Comment by zts59** [comment]
后面用DOCKER布署同样的任务,导致容器启动不了了

**Comment by zts59** [comment]
上述同步复制任务时，WIN10系统，16G内存，ALIST占用需要优化，在本机浏览localhost:5244/@manage/tasks/copy，alist占用内存不断增加，此时浏览器的占用内存也会不断增加，最后会爆掉，建议对任务需要分批执行：计划任务和正执行的中，设置任务进程数，划分多少个任务为一个计划等等。

**Comment by dreamoeu** [comment]
你将config.json里面task_persistant值改为false试试，~~大概可以让data.db不无限膨胀吧~~
```json
  "tasks": {
    "download": {
      "workers": 5,
      "max_retry": 1,
      "task_persistant": false
    },
    "transfer": {
      "workers": 5,
      "max_retry": 2,
      "task_persistant": false
    },
    "upload": {
      "workers": 5,
      "max_retry": 0,
      "task_persistant": false
    },
    "copy": {
      "workers": 5,
      "max_retry": 2,
      "task_persistant": false
    }
  },
```

**Comment by pongfcnkl** [comment]
可以用rclone管理复制任务
比如 rclone copy alist:百度 alist:阿里 ,...等参数进行复制，不太会卡住。
